apps/mark-scan/accessible-controller/src/port.rs

```diff
@@ -95,8 +95,17 @@ impl io::Read for Port {
 impl fmt::Debug for Port {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         f.debug_struct("Port")
-            .field("name", &self.inner.name())
-            .field("baud_rate", &self.inner.baud_rate())
+            .field(
+                "name",
+                &self.inner.name().unwrap_or_else(|| "n/a".to_string()),
```

Comment:
I know your previous commit removed usage of `unwrap` but IIUC that was for better error handling. In this case I don't think we want to propagate the error up, since the name of the port is somewhat inconsequential but we need to provide a `None` alternative.